index.src.html

```diff
@@ -240,53 +240,40 @@ <h3 id="minimum-data">
   bare minimum necessary to achieve the desired use cases?  If not, why not and
   why expose the additional information?
 
-  <h3 id="pii">
-    How does the specification deal with personal information allowing to
-    single out the user?
+  <h3 id="personal-data">
+    Does this specification deal with personal information or
```

Comment:
This isn't useful. I've seen a lot of replied to this quesiton as simple "No". We'd like to know "how", as to suggest a thought process ;-) "No" is not the answer.

Reply:
Is the suggestion to change the question to "How does this specification deal with personal information or personally-identifiable information or information derived thereof?"

Reply:
Yes, unless you have got a rationale for using "does". My fear is that though we explain in the document that we ask for explanations, the use of "does" invites the answer "No" (or "Yes"). Answering "No" to "Does" is a bit more difficult ;)

Reply:
See 
https://github.com/w3ctag/security-questionnaire/pull/41/commits/4310f1db6d8c8d94b05f1a2ae14f3d2271b8a1ce